src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -436,18 +460,10 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
                 answerStrings.remove(otherAnswer);
             }
 
-            for (String answerString : answerStrings) {
-                if (answerString.isEmpty()) {
-                    continue;
-                }
-
+            int numChoices = getNumberOfEmptyResponseOfQuestion(answerStrings, answerFrequency);
```

Comment:
Similarly here. You're assigning the "number of empty response" to "number of choices".